packages/code-du-travail-data/dataset/fiches_ministere_travail/index.js

```diff
@@ -12,6 +12,13 @@ function parseDom(dom, url) {
     .map(n => n.textContent.trim())
     .filter(t => t !== "L’INFO EN PLUS" && t !== "POUR ALLER PLUS LOIN");
 
+  const ariane = $(dom.window.document, "nav.page__breadcrumb")
```

Comment:
could be better to target `nav.page__breadcrumb a` so you don't need to remove `\n` and `>`

Reply:
```js
$$(dom.window.document, "nav.page__breadcrumb").slice(1).map(el => el.textContent.trim())